installer/installation-notes.md

```diff
@@ -0,0 +1,135 @@
+    will need to manually inject the TLS certificate into the squareone
+    configuration post-installation.
+
+  * For each of the second and third, which are the cases the installer
+    script is designed to support, the RSP manages the top-level
+    ingress, and Let's Encrypt is used to manage TLS certificates.  The
+    difference between these cases is that one has a pre-defined fixed
+    IP address (e.g. the T&S sites), and one uses whatever the
+    Kubernetes provider gives up (e.g. GKE).
+
+  * In the second case, now is a good time to make sure that the top-level
+    endpoint (e.g. "tucson-teststand.lsst.codes") resolves to the
+    correct IP address, whether directly as an A record or as a CNAME to
+    an A record.
+
+  * For the third case you will set up the DNS record once you know the
+    IP address of the endpoint.
+
+If this is the first time you've installed the RSP in a given domain,
+and you are managing TLS certs via Let's Encrypt for the RSP instance,
+you will need to put a glue record into that domain to allow Let's
+Encrypt to work its magic.  This process is documented at
+https://phalanx.lsst.io/ops/cert-issuer/route53-setup.html for Route
+53.  If you're not using Route 53, you're on your own, but use that
+document as a guide for the tasks you will need to perform.
+
+## Pre-flight check
+
+Check the rest of the phalanx definitions.  Particular items of
+ interest:
+ 
+   * NFS mountpoints in nublado2 and moneypenny
+   
+   * If the Firefly portal has a replicaCount greater than one, it is
+     imperative that `firefly_shared_workdir` be set and that it reside
+     on an underlying filesystem that supports shared multiple-write.
+     At GKE, this is currently Filestore (therefore NFS), and at NCSA,
+     it is currently a hostPath mount to underlying GPFS.  Currently the
+     provisioning of this underlying backing store is manual, so make
+     sure you either have created it, or gotten a system administrator
+     with appropriate permissions for your site to do so.  The default
+     UID for Firefly is 91, although it is tunable in the deployment if
+     need be.
+     
+   * For T&S sites that require instrument control, make sure you have
+     any Multus network definitions you need for nublado2.
+
+## Perform the installation
+
+`./install.sh <environment> <token>`
+
+(where "environment" looks like "tucson-teststand"--it must match a
+filename in ../science-platform/values-<environment>.yaml, and "token"
+is the Vault write token you got out of 1Password.
+
+### Set up DNS record for dynamically-assigned ingress
+
+If you are relying on the Kubernetes hosting machinery to assign an IP
+address for your endpoint, while the installer is running, wait until
+the ingress-nginx-controller service comes up and has an external IP
+address; then go set the A record for your endpoint to that address (or
+set an A record with that IP address for the ingress and a CNAME from
+the endpoint to the A record).
```

Comment:
May be worth saying that if this is a long-lived installation, you probably then want to promote this IP to a static IP.